README.md

```diff
@@ -57,17 +59,21 @@ For example, `tox -e py37-unit` will run the unit tests for python 3.7
 To see all the available options, run `tox -l`.
 
 ## Releasing
+
 To create a new releaes go to Releases page, press 'Draft a new release', create a tag
-with a version you want to be released, fill the release notes and press 'Publish release'. 
+with a version you want to be released, fill the release notes and press 'Publish release'.
 Github actions will take care of publishing it to PyPi.
 
 ## Responsible Disclosure
+
 If you have any security issue to report, contact project maintainers privately.
 You can reach us at <mailto:github@amplify.com>
 
 ## Contributing
+
 We welcome pull requests! For your pull request to be accepted smoothly, we suggest that you:
- 1. For any sizable change, first open a GitHub issue to discuss your idea.
- 2. Create a pull request.  Explain why you want to make the change and what it’s for.
+
+1. For any sizable change, first open a GitHub issue to discuss your idea.
+1. Create a pull request.  Explain why you want to make the change and what it’s for.
```

Comment:
```suggestion
2. Create a pull request.  Explain why you want to make the change and what it’s for.
```

Reply:
Fix it in a different way. Changed it to unordered list, because mdformat fails on this case.